server/views/partials/analytics/chartFeedbackForm.njk

```diff
@@ -2,14 +2,13 @@
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/textarea/macro.njk" import govukTextarea %}
 
-{# Required variables: graphId, csrfToken #}
-{# Optional variables: form (if there are errors) #}
+{# Required variables: chartFeedbackForm, csrfToken #}
 
-{% macro commentTextarea(graphId, id) %}
+{% macro commentTextarea(chartFeedbackForm, id) %}
   {{ govukTextarea({
-    id: graphId + "-" + id + "Comments",
+    id: chartFeedbackForm.formId + "-" + id + "Comments",
     name: id + "Comments",
-    value: form.data[id + "Comments"] | default(""),
+    value: chartFeedbackForm.getField(id + "Comments").value,
```

Comment:
Just double checking, when the value is not in the form data `.value` would be `undefined`, but this is fine because `govukTextarea()` would just ignore it right? It's not gonna be converted to `'undefined'` right?